packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Snippets/Snippet.cs

```diff
@@ -143,5 +166,9 @@ public static ValueExpression Invoke(this PropertyProvider property,
 
         public static ScopedApi<bool> NotEqual(this ParameterProvider parameter, ValueExpression other)
             => new BinaryOperatorExpression("!=", parameter, other).As<bool>();
+
+        public static CodeWriterDeclaration Declaration(this ParameterProvider variableExpression) => ((VariableExpression)variableExpression).Declaration;
```

Comment:
Why do we need this method? Can't we just call AsExpression().Declaration?

Reply:
removed